lib/widgets/in_game_app_bar.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg_provider/flutter_svg_provider.dart';
+import 'package:have_you_heard/constants/colors.dart';
+
+class InGameAppBar extends StatelessWidget implements PreferredSizeWidget {
+  @override
+  final Size preferredSize;
+  final String title;
+  final VoidCallback? onSettingsPressed;
```

Comment:
I think we can default to going to the Setting page `Get.toNamed(SettingsScreen.route)`?